### PR TITLE
XW-1207 | Remove obsolete abstract logic for Krux

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1207

## Description

1) remove Krux tracker abstraction.
2) notify console when actual track occurs, and not when it's called.

## Reviewers

@Wikia/x-wing @dianafa @nandy-andy

### CODE REVIEW

**Comment by vforge** [comment]
I think it's complete now, smoke tested. Pinging @nandy-andy and @dianafa for additional LGTM.

File: front/common/modules/Ads.js [hunk 1]
```diff
@@ -127,15 +126,19 @@ class Ads {
...
 	 * @returns {void}
 	 */
-	kruxTrackFirstPage() {
-		this.kruxTracker.trackPageView();
+	trackKruxPageView() {
+		if (this.krux && typeof this.krux.load === 'function') {
+			console.info('Track pageView: Krux');
+
+			this.krux.load(M.prop('tracking.krux.mobileId'));
+		}
```
Comment (nandy-andy) [inline front/common/modules/Ads.js]: Can we log to Kibana for example to see how many times it fails to load krux?

Comment (rogatty) [inline front/common/modules/Ads.js]: Currently we don't have a way for logging front end errors, it all goes to console only.
There is a spike that you can bother POs about, though: https://wikia-inc.atlassian.net/browse/XW-123

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by dianafa** [comment]
LG :+1:

**Pull request merged**